backend/src/schema/types/type/Badge.gql

```diff
@@ -3,8 +3,6 @@ type Badge {
   type: BadgeType!
   status: BadgeStatus!
   icon: String!
-  #createdAt: DateTime
-  #updatedAt: DateTime
```

Comment:
I understand why you want to clean this up, but you haven't lost hope on refactoring to use `DateTime`, right?
have you seen https://github.com/neo4j-graphql/neo4j-graphql-js/issues/272 ??